applications/GeoMechanicsApplication/custom_elements/Policies.md

```diff
@@ -0,0 +1,137 @@
+
+For the 3D stress state, the strain vector is defined as:
+```math
+\vec{\epsilon} = \begin{bmatrix} \epsilon_{xx} \\
+                                 \epsilon_{yy} \\
+                                 \epsilon_{zz} \\
+                                 \epsilon_{xy} \\
+                                 \epsilon_{yz} \\
+                                 \epsilon_{xz} \end{bmatrix}
+```
+
+For the plane strain stress state, it is defined as:
+```math
+\vec{\epsilon} = \begin{bmatrix} \epsilon_{xx} \\
+                                 \epsilon_{yy} \\
+                                 0 \\
+                                 \epsilon_{xy} \end{bmatrix}
+```
+
+Lastly, for the axisymmetric stress state, it is defined as:
+```math
+\vec{\epsilon} = \begin{bmatrix} \epsilon_{xx} \\
+                                 \epsilon_{yy} \\
+                                 u_r/r \\
+                                 \epsilon_{xy} \end{bmatrix}
+```
+Where $u_r$ is the radial displacement and $r$ is the radial coordinate. 
+
+### B-matrix
+The B matrix is used to relate strains and displacements. Therefore, its elements are filled with the spatial gradients of the shape functions ($N$).
+
+For a 3D element consisting of 4 nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y & \delta N_1/\delta z\\
+\delta N_2/\delta x & \delta N_2/\delta y & \delta N_2/\delta z\\
+\delta N_3/\delta x & \delta N_3/\delta y & \delta N_3/\delta z\\
+\delta N_4/\delta x & \delta N_4/\delta y & \delta N_4/\delta z
+\end{bmatrix}
+```
+Resulting in the following B-matrix:
+```math
+B =
+\begin{bmatrix}
+\delta N_1/\delta x & 0 & 0 & && & \delta N_4/\delta x & 0 & 0 \\
+0 & \delta N_1/\delta y & 0 & && & 0 & \delta N_4/\delta y & 0\\
+0 & 0 & \delta N_1/\delta z & &\dots& & 0 & 0 & \delta N_4/\delta z\\
+\delta N_1/\delta y & \delta N_1/\delta x & 0 & && & \delta N_4/\delta y & \delta N_4/\delta x & 0 \\
+0 & \delta N_1/\delta z & \delta N_1/\delta y & && & 0 & \delta N_4/\delta z & \delta N_4/\delta y \\
+\delta N_1/\delta z & 0 & \delta N_1/\delta x & && & \delta N_4/\delta z & 0 & \delta N_4/\delta x
+\end{bmatrix}
+```
+Where the first three columns are for the first node, the next three columns for the second node and so on for the number of nodes in the element.
+
+For a 2D element consisting of three nodes, the gradient looks like:
+```math
+\nabla{N} =
+\begin{bmatrix}
+\delta N_1/\delta x & \delta N_1/\delta y \\
+\delta N_2/\delta x & \delta N_2/\delta y \\
+\delta N_3/\delta x & \delta N_3/\delta y
+\end{bmatrix}
+```
+Where the rows depict the different nodes in the element (i.e. $N_i$ is the shape function for node $i$). For the plane strain case, this would result in the following B-matrix
```

Comment:
```suggestion
Where the rows depict the different nodes in the element (i.e. $N_i$ is the shape function for node $i$). For the plane strain case, this would result in the following B-matrix:
```